.eslintrc.js

```diff
@@ -20,9 +20,7 @@ module.exports = {
   },
   "globals": {
     "$": true,
-    "jQuery": true,
-    "React": true,
-    "ReactDOM": true
+    "jQuery": true
```

Comment:
I'd _really_ like to do this for jQuery too, but that's a big enough task that I'm saving it for another PR.